WordPressKit/HTTPClient.swift

```diff
@@ -7,6 +7,18 @@ struct HTTPAPIResponse<Body> {
     var body: Body
 }
 
+extension HTTPAPIResponse where Body == Data {
+    var bodyText: String? {
+        var encoding: String.Encoding?
```

Comment:
Nitpick. I think this should work, too:

```suggestion
        let encoding: String.Encoding?
```

No difference at runtime, I don't think, but simply the "best practice" to prefer immutable values whenever possible.

Reply:
Making it a `let` would require us adding a `else` case below and assigning it a nil value. The additional code doesn't seem to be worth it 😄